README.md

```diff
@@ -1,66 +1,109 @@
--------
-### ⚠️  PLEASE READ THE [INSTRUCTIONS](/INSTRUCTIONS.md) FOR GUIDELINES ON HOW TO START YOUR PACKAGE.
-> Don't forget to remove this warning while updating this README.
--------
-
-# {package-name}
+# next-button
 
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/next-button
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/next-button.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/next-button.svg
+[build-status-url]:https://github.com/moxystudio/next-button/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/next-button/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/next-button
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-button/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/next-button
+[david-dm-image]:https://img.shields.io/david/moxystudio/next-button.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/next-button?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/next-button.svg
 
-{package-drescription}
+A react button that doubles down as a link.
```

Comment:
I would change to "A React button which can behave as a link"

Reply:
I don't like the `behave as a link` because it is indeed a link, it doesn't just behave as one. I'll leave this as is.